### PR TITLE
CDATA Fix

### PR DESCRIPTION
**Pull request by jeffreypicard**
Fix the URI fields to use CDATA as per the spec.
Added a CDATA struct for the cases where the URI field isn't
an unnamed field within another with attributes.

### CODE REVIEW

**Comment by jeffreypicard** [comment]
The cdata attribute for xml was introduced in 1.6 I think, so this PR breaks backwards compatibility :P

**Comment by rs** [comment]
@kalbasit please review

**Comment by kalbasit** [comment]
on it

File: vast.go [hunk 1]
```diff
@@ -13,7 +13,11 @@ type VAST struct {
 	Ads []Ad `xml:"Ad"`
 	// Contains a URI to a tracking resource that the video player should request
 	// upon receiving a “no ad” response
-	Errors []string `xml:"Error"`
+	Errors []CDATA `xml:"Error"`
```
Comment (kalbasit) [inline vast.go]: You missed the `Errors` in the InLine struct https://github.com/MadHive/vast/blob/4a7b87d90d554b07cc3bad8bfcf43718f10c7256/vast.go#L69

**Comment by kalbasit** [comment]
@jeffreypicard please address my comment and rebase over master.

**Comment by kalbasit** [comment]
@jeffreypicard can you please address my comments so we can merge this PR? Thank you.

**Pull request closed without merging**